src/tests/screens/SurveyDetail/detail.test.js

```diff
@@ -8,14 +8,22 @@ import { Router } from 'react-router-dom';
 import { DetailsContext } from 'contexts/details';
 import { SurveyStatusProvider } from 'contexts/surveyStatus';
 import Detail from 'screens/SurveyDetail/detail';
+import buildQuestionList from 'screens/SurveyDetail/QuestionBuilders/buildQuestionList';
 import SurveyDetailResponse from 'tests/fixtures/surveyDetailResponse.json';
 
 describe('When Detail component is mounted', () => {
+  const questionList = buildQuestionList(SurveyDetailResponse);
+
   it('shows the survey image', () => {
     const history = createMemoryHistory();
     const { container } = render(
       <Router history={ history }>
-        <DetailsContext.Provider value={ SurveyDetailResponse.data }>
+        <DetailsContext.Provider
+          value={{
+            surveyDetail: SurveyDetailResponse,
+            questionList: questionList
```

Comment:
How about added the [object-shorthand](https://eslint.org/docs/rules/object-shorthand) rule to ESlint too? 💭 
```suggestion
            questionList
```

Reply:
@carryall fixed reviews in [b93b1a3](https://github.com/rafayet-monon/surveyor/pull/28/commits/b93b1a3c57c4fd3bc24715e1093d6ef291fbbe7a)